synapseutils/copy.py

```diff
@@ -185,6 +185,7 @@ def _copyRecursive(syn, entity, destinationId, mapping=None, skipCopyAnnotations
         raise ValueError("Excluded types can only be a list of these values: file, table, and link") 
 
     ent = syn.get(entity, downloadFile=False)
```

Comment:
replace this with a get entity bundle, which removes need for two more rest calls later on

Reply:
note: requesting access_requirements in a bundle is deprecated and will always return an empty list